### ISSUE TITLE
Video resolution seems reduced by force on 1.0.2

### ISSUE THREAD

**Issue by cha2hyun** [EthanArbuckle/carplay-cast#79]
## Tested Device 
- iPhone X / ios 14.6 / checkrain
- iPhone 12 mini / ios 14.6 / uncover 

## Tested Car
- 2022 BMW IX3
- 2021 Jaguar F Pace
- 2020 Chevorlet Bolt EV

## Version 
- 1.0.2

## Issue
There is a problem with the video resolution. The width seems to have been reduced by force. Video is not played as original resolution (16:9) it seems like 1:1.5 (?) on full-screen. 

The video player i tested are Youtube (latest version), and DS video 

Maybe auto hide dock cause resolution problem.

**Comment by cha2hyun** [comment]
## Original Size
![스크린샷 2023-02-09 오전 10 44 15](https://user-images.githubusercontent.com/56015532/217694722-1aa005a1-8a6f-40c9-9d13-b9266e8dc222.png)

## Reduced Size on full screen
![스크린샷 2023-02-09 오전 10 44 20](https://user-images.githubusercontent.com/56015532/217694740-fbd39e98-6a97-4762-96eb-8ea5805574bb.png)

## Extra
1. Horizontal mode shows correct resolution.
2. I tested my own video which is made of 16:9 resolution, but it seems 1:1.5(?) on full-screen 
3. version 1.0.1 has same problem

**Comment by EthanArbuckle** [comment]
You're right that autohiding dock could influence scaling. I'm thinking it may be best to add options that allow you to manually modify the scaling parameters. This puts less burden on getting a "one size fits all" solution right, at the cost of users potentially needing to make some adjustments themselves. Any objections?
